src/openrct2/ride/Ride.cpp

```diff
@@ -3091,7 +3092,7 @@ static constexpr const CoordsXY word_9A2A60[] = {
  */
 static Vehicle* vehicle_create_car(
     RideId rideIndex, int32_t carEntryIndex, int32_t carIndex, int32_t vehicleIndex, const CoordsXYZ& carPosition,
-    int32_t* remainingDistance, TrackElement* trackElement)
+    int32_t* remainingDistance, TrackElement* trackElement, bool isReversed)
```

Comment:
Its probably best to change this entire function signature to pass Ride* as its always obtained prior to calling this, you don't need to pass isReversed since you can get the mode from the ride.

Reply:
Done